planetarium/NineChronicles
# [WIP] Upgrade unity version 2020.3.0f1 LTS

### Description

1. Upgrade unity version to 2020.3.0f1

### How to test

1. Open this project with unity 2020.3.0f1 LTS.
2. Check there is no errors on compile and build processes.

### Related Links

1. [유니티 업그레이드](https://app.asana.com/0/958521740385861/1200074654976636/f)
